scripts/common.sh

```diff
@@ -0,0 +1,15 @@
+NODE1_NAME=node1
+NODE1_UUID=16e7d8a7-bfa9-428b-9117-363341bb330b
```

Comment:
Do they cause any problems? I think it could still be useful to have consistent UUIDs, even if they're not used in examples.